src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const str = sourceString;
+  const result = {};
+  const items = str.split(';').filter((item) => item.trim());
```

Comment:
The `filter((item) => item.trim())` is used to remove empty items after splitting the string. This is a good practice to ensure that empty strings do not result in undefined keys or values in the result object.